android/java/res/values/brave_colors.xml

```diff
@@ -428,16 +428,20 @@
     <color name="location_bar_background_color_for_ntp">#f2f2f2</color>
     <color name="toolbar_background_color_for_ntp">@android:color/white</color>
 
-    <color name="primitive_blurple_10">#0f0663</color>
+    <color name="primitive_blurple_10">#000668</color>
```

Comment:
These colors are used in other UIs as well, did you check how they look with the change?

Reply:
Yep, they are almost identical
<img width="140" height="140" alt="Screenshot from 2025-08-08 15-44-58" src="https://github.com/user-attachments/assets/e1c06eaa-92e2-4dcc-9ab4-ab40973e6d04" />
<img width="148" height="139" alt="Screenshot from 2025-08-08 15-44-43" src="https://github.com/user-attachments/assets/bb6f244c-a965-4120-adda-0b4b8a6a3c09" />